### PR TITLE
RHPROD-103 Switch to using unirest to avoid spurious ProdSec API erro…

### PR DESCRIPTION
**Pull request by rnc**
…rs. Clean up test code

### CODE REVIEW

**Comment by rnc** [comment]
cc @jeremychoi

File: src/test/java/com/redhat/engineering/srcclr/internal/SecurityDataProcessorTest.java [hunk 1]
```diff
@@ -242,7 +237,7 @@ private Object executeMethod( Object instance, String name, Object[] params ) th
         for ( int i = 0; i < params.length; i++ )
             types[i] = params[i].getClass();
 
-        Method m = c.getDeclaredMethod( name, types );
+        Method m = c.getDeclaredMethod( "lookUpAPI", types );
```
Comment (jeremychoi) [inline src/test/java/com/redhat/engineering/srcclr/internal/SecurityDataProcessorTest.java]: This change is also fine. Just to clarify, the reason for getting the method name parameter was that it was to be used for multiple tests, although there's only one test at the moment. Maybe I could change it back later if more test cases are needed.

**Comment by jeremychoi** [comment]
LGTM. thanks for making it more stable.

File: src/test/java/com/redhat/engineering/srcclr/internal/SecurityDataProcessorTest.java (hunk 1 above)
Comment (rnc) [reply src/test/java/com/redhat/engineering/srcclr/internal/SecurityDataProcessorTest.java]: Ah ok. Yes that makes sense to me :-)

**Pull request merged**